docs/development/workflow/git_edit_workflow_examples.rst

```diff
@@ -75,16 +75,24 @@ Set up an isolated workspace
 
   If you are using a different distribution, see :ref:`virtual_envs` for
   instructions for creating and activating a new environment.
-+ Install our branch in this environment (will not work in python 3) with::
-
-    python setup.py develop
++ Install our branch in this environment with::
+  
+    pip install -e .
+    
+  Alternatively, use the `astropy/setup.py <https://github.com/astropy/astropy/blob/master/setup.py/>`_
```

Comment:
Again, I wonder if we should give this at all (I know that I use it, but I probably should change too!)

Reply:
Yup, agreed. Was trying to be helpful but may not be necessary in this context.